library/src/com/google/maps/android/geojson/GeoJsonLineStringStyle.java

```diff
@@ -0,0 +1,170 @@
+package com.google.maps.android.geojson;
+
+import com.google.android.gms.maps.model.PolylineOptions;
+
+import java.util.Observable;
+
+/**
+ * A class that allows for GeoJsonLineString objects to be styled and for these styles to be
+ * translated into a PolylineOptions object. {@see
+ * <a href="https://developer.android.com/reference/com/google/android/gms/maps/model/PolylineOptions.html">
+ * PolylineOptions docs</a> for more details about the options.}
+ */
+public class GeoJsonLineStringStyle extends Observable implements GeoJsonStyle {
+
+    private final static String GEOMETRY_TYPE_REGEX
```

Comment:
Leaving it as a String for now. @amorris13 previous suggestion was to make it an array of strings.